interface/CMakeLists.txt

```diff
@@ -18,6 +18,9 @@ if (TFM_MULTI_CORE_TOPOLOGY)
     add_library(platform_ns STATIC)
 endif()
 
+#CMSIS
+include_directories(${TFM_TEST_REPO_PATH}/CMSIS/RTOS2/Include)
```

Comment:
Not a major problem, but this would require the tf-m-tests paths to be set, even when not building tests. Okay since it was done this way before.

Reply:
`TFM_TEST_REPO_PATH` is set by configure command by TF-M...

Reply:
This is my point - somehow we need to configure with `TFM_TEST_REPO_PATH` even when we don't build tests... But it's okay since the CMSIS headers are there...